src/organism.jl

```diff
@@ -0,0 +1,24 @@
+
+
+Base.@kwdef struct OrganismalPars{F}
```

Comment:
we should discuss the issue of these changing behaviourally, e.g. colour change - does this mean they need to be mutable?

Reply:
I was thinking about that too. Most of organismal params can change through a simulation. I tried to put most of the params I thought could change in OrganismalVars, because I understood that those get rebuilt in each time step of a simulation.